src/plugins/presentation_util/public/components/controls/control_group/editor/manage_control.tsx

```diff
@@ -0,0 +1,149 @@
+} from '@elastic/eui';
+
+import { ControlGroupStrings } from '../control_group_strings';
+import { ControlEditorComponent, ControlWidth } from '../../types';
+import { CONTROL_WIDTH_OPTIONS } from '../control_group_constants';
+
+interface ManageControlProps {
+  title?: string;
+  onSave: () => void;
+  width: ControlWidth;
+  onCancel: () => void;
+  removeControl?: () => void;
+  controlEditorComponent?: ControlEditorComponent;
+  updateTitle: (title: string) => void;
+  updateWidth: (newWidth: ControlWidth) => void;
+}
+
+export const ManageControlComponent = ({
+  controlEditorComponent,
+  removeControl,
+  updateTitle,
+  updateWidth,
+  onCancel,
+  onSave,
+  title,
+  width,
+}: ManageControlProps) => {
+  const [currentTitle, setCurrentTitle] = useState(title);
+  const [currentWidth, setCurrentWidth] = useState(width);
+
+  const [controlEditorValid, setControlEditorValid] = useState(false);
+  const [editorValid, setEditorValid] = useState(false);
+
+  useEffect(() => setEditorValid(Boolean(currentTitle)), [currentTitle]);
+
+  return (
+    <>
+      <EuiFlyoutHeader hasBorder>
+        <EuiTitle size="m">
+          <h2 id="flyoutTitle">{ControlGroupStrings.manageControl.getFlyoutTitle()}</h2>
+        </EuiTitle>
+      </EuiFlyoutHeader>
+      <EuiFlyoutBody>
+        <EuiForm>
+          <EuiFormRow label={ControlGroupStrings.manageControl.getTitleInputTitle()}>
+            <EuiFieldText
+              placeholder="Placeholder text"
+              value={currentTitle}
+              onChange={(e) => {
+                updateTitle(e.target.value);
+                setCurrentTitle(e.target.value);
+              }}
+              aria-label="Use aria labels when no actual label is in use"
+            />
+          </EuiFormRow>
+          <EuiFormRow label={ControlGroupStrings.manageControl.getWidthInputTitle()}>
+            <EuiButtonGroup
+              buttonSize="compressed"
+              legend={ControlGroupStrings.management.controlWidth.getWidthSwitchLegend()}
+              options={CONTROL_WIDTH_OPTIONS}
+              idSelected={currentWidth}
+              onChange={(newWidth: string) => {
+                setCurrentWidth(newWidth as ControlWidth);
+                updateWidth(newWidth as ControlWidth);
+              }}
+            />
+          </EuiFormRow>
+
+          <EuiSpacer size="l" />
+          {controlEditorComponent &&
+            controlEditorComponent({ setValidState: setControlEditorValid })}
+          <EuiSpacer size="l" />
+          {removeControl && (
+            <EuiButton
+              aria-label={`delete-${title}`}
+              iconType="trash"
+              color="danger"
+              onClick={() => {
+                onCancel();
+                removeControl();
+              }}
+            >
+              {ControlGroupStrings.management.getDeleteButtonTitle()}
+            </EuiButton>
+          )}
+        </EuiForm>
+      </EuiFlyoutBody>
+      <EuiFlyoutFooter>
```

Comment:
Not certain of the exact fix, but this footer is still floating up from the bottom of the view:
<img width="726" alt="Screen Shot 2021-09-29 at 2 55 52 PM" src="https://user-images.githubusercontent.com/446285/135340547-87a7940d-de13-4c15-8479-47f846048985.png">

Reply:
I think this is a side-effect of my stub / total re-implementation of the overlays service. My hope is that this issue only happens in storybook, and as soon as we put it into Kibana this will disappear. If not we can revisit